src/opentelemetry_exporter.erl

```diff
@@ -179,6 +182,12 @@ is_proplist([]) -> true;
 is_proplist([{K,_}|L]) when is_atom(K) or is_binary(K) -> is_proplist(L);
 is_proplist(_) -> false.
 
+encode_key(Key) when is_atom(Key) -> erlang:atom_to_binary(Key, unicode);
+encode_key(Key) -> Key.
+
+encode_value(Value) when is_atom(Value) -> erlang:atom_to_binary(Value, unicode);
```

Comment:
Why are there two functions that do the same thing? Should just have a `to_binary` function.